packages/server/src/utils/builders/compose.ts

```diff
@@ -188,6 +192,35 @@ const createEnvFile = (compose: ComposeNested) => {
 	writeFileSync(envFilePath, envFileContent);
 };
 
+export const processComposeFile = async (compose: ComposeNested) => {
+	const { COMPOSE_PATH } = paths();
+	const command = getProcessComposeFileCommand(compose);
+
+	if (compose.serverId) {
+		await execAsyncRemote(compose.serverId, command);
+	} else {
+		await execAsync(command, {
+			cwd: join(COMPOSE_PATH, compose.appName, "code"),
+		});
+	}
```

Comment:
Is not necessary to handle the server case here, because when we execute on the dokploy server it runs the `buildCompose` FN that means it will never execute on the remote server, for remote servers we use `getBuildComposeCommand` FN, so you can safely remove this validation and just invoke the execAsync